term.go

```diff
@@ -286,7 +287,7 @@ func (t *Terminal) run() {
 			fyne.LogError("pty read error", err)
 		}
 
-		t.handleOutput(buf[:num])
+		leftOver = t.handleOutput(append(leftOver, buf[:num]...))
```

Comment:
As leftover will normally be empty, wouldn't it be more efficient to do this append in an if block before the handle output?

Reply:
That's true.  I'll add in the if.